### ISSUE TITLE
ES6 default import does not work

### ISSUE THREAD

**Issue by tmxyz** [GoogleChrome/chrome-launcher#37]
Hi,

### Problem
I was trying to import the `chrome-launcher ` using default import syntax like that:
```
import chromeLauncher from 'chrome-launcher';
```
However, when I tried using it I got an error messageL `ReferenceError: chromeLauncher is not defined`.

### Workaround
Importing all of the modules contents works.
```
import * as chromeLauncher from 'chrome-launcher';
```

I suggest either explicitly stating in the documentation that the modules has to be imported with import all contents syntax or add support for the default import.

**Comment by samccone** [comment]
Hello and thanks for reaching out.

looking here at the code, we have no default export 
https://github.com/GoogleChrome/chrome-launcher/blob/master/chrome-launcher.ts#L85
thus this is working as implemented.

Did you see a code example that made you think you could import using the default syntax? If so we should fix that.

Thanks so much for taking time out of your day to open a bug!

**Comment by tmxyz** [comment]
The documentation uses the require pattern, so there there is nothing wrong with it, but people using ES6 imports (my case) have to guess (or just check the source code 😄 ) which pattern to use. Since default import is more common it was the behaviour I was expecting.

**Comment by samccone** [comment]
More than happy to land a PR adding it as the default export and or making documentation change to make this explicit 

Thanks again!

**Comment by paulirish** [comment]
shipped in 0.10.0

**Comment by adamhenson** [comment]
It would be great if the documentation could be updated to illustrate an example with "ES6 Modules". I'd put in a PR myself, but I actually wasn't able to get this to work based on the [documentation using `require`](https://github.com/GoogleChrome/chrome-launcher#launching-chrome). I started digging in the source code a little and realized we now have a class that would need to be instantiated (shown below). I didn't really have time to get too deep, so I ended up just using `require`.

```javascript
import ChromeLauncher from 'chrome-launcher';

// ...

// undocumented implementation
const chromeLauncher = new ChromeLauncher(options);

// ...

// this is returning `undefined`
const chrome = await chromeLauncher.launch();

// and obviously this isn't working either...
// Cannot read property 'kill' of undefined
await chrome.kill();
```
